lib/entities/webhook.ts

```diff
@@ -28,33 +28,36 @@ export type WebhookProps = {
   /**
    * Username for basic http auth
    */
-  httpBasicUsername: string
+  httpBasicUsername: string | null
 
   /**
    * Password for basic http auth
    */
-  httpBasicPassword: string
+  httpBasicPassword: string | null
 
   /**
    * Headers that should be appended to the webhook request
    */
-  headers: {
-    [key: string]: string
-  }
+  headers: Array<{ key: string; value: unknown; secret?: boolean }>
 
   /**
    * Topics the webhook wants to subscribe to
    */
   topics: string[]
 
+  /**
+   * Webhook filters
+   */
+  filters?: Array<{ [key: string]: any }>
+
   /**
    * Transformation to apply
    */
   transformation?: {
     method?: string
     contentType?: string
     includeContentLength?: boolean
-    body?: object
+    body?: Record<string, any>
```

Comment:
`body` can be any valid JSON, so `true`, `['test']`, or `123` are valid too

Reply:
updated

Reply:
shouldn't that be of type 

```ts
body?: JsonValue
```

Reply:
I obviously commented on an outdated version 😆 👍